guides/v2.1/cloud/cdn/fastly-vcl-wordpress.md

```diff
@@ -1,7 +1,7 @@
 ---
 group: cloud-guide
 subgroup: 090_configure
-title: Custom redirect to Wordpress VCL
+title: Set up redirects to WordPress using Fastly
```

Comment:
This file seems to have similar issues from the referrer content, so I won't continue to duplicate comments, but you can review based on that.